### PR TITLE
Milestones for 7/25

### PR DESCRIPTION
**Pull request by zaaakk**

### CODE REVIEW

File: rome_app/models.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def prints(self):
     def format_trp_id(self, trp_id):
         return '%04d' % int(trp_id)
 
+    def related_essays(self):
+        print self.essay_set.all()
```
Comment (bcail) [inline rome_app/models.py]: remove print statement

File: rome_app/templates/rome_templates/page_detail.html [hunk 2]
```diff
@@ -159,8 +159,12 @@
     h2 {
         font-weight: 300;
         margin-top:4px;
+        font-size:14px;
     }
 
+
+
+
```
Comment (bcail) [inline rome_app/templates/rome_templates/page_detail.html]: remove extra blank lines

File: rome_app/views.py [hunk 3]
```diff
@@ -598,12 +600,14 @@ def essay_detail(request, essay_slug):
     related_list=[]
     for work in essay.related_works():
         current_work={}
-        current_work['title']=work['primary_title']
-        current_work['creator']=work.get('creator')
+        current_work['title']=work['primary_title'].encode('ascii', 'ignore').decode('ascii')
```
Comment (bcail) [inline rome_app/views.py]: These lines where you're encoding the data and then decoding it again - I don't think you need to do that. In related_works(), the data comes through r.json(). I just looked at that, and it looks like that data is unicode already. So you should be able to just use work['primary_title'], and leave the rest of the line off.

If you had an issue somewhere, though, where that didn't work, let me know and we can try to address it.

Comment (zaaakk) [inline rome_app/views.py]: I was having an issue with stripping some of the encoding, but I just removed the encoding and decoding and it appears to be fine without that.

**Pull request merged**